src/frontend/NavigationStacks/AppStack.tsx

```diff
@@ -26,6 +26,8 @@ import { LeaveProjectScreen } from "../screens/LeaveProject/LeaveProject";
 import { LeaveProjectProgress } from "../screens/LeaveProject/LeaveProjectProgess";
 import { LeaveProjectCompleted } from "../screens/LeaveProject/LeaveProjectCompleted";
 import { AlreadyOnProj } from "../screens/AlreadyOnProject";
+import UnableToLeaveScreen from "../screens/UnableToLink";
```

Comment:
Should the component name be `UnableToLinkScreen`? (also might as well convert to a named export too)